.config/cargo_spellcheck.dic

```diff
@@ -0,0 +1,75 @@
+80
```

Comment:
Are all these words required? We should not add words that are not and keep this list as small as possible. Some of them are kinda familiar to me in a way ... ^^